### PR TITLE
Support concurrent replicas

### PR DESCRIPTION
**Pull request by johnwatson484**
- Support concurrent replicas
- Fix syntax of insights
- Add transaction lock

### CODE REVIEW

File: changelog/db.changelog-2.6.xml [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<?xml version="1.1" encoding="UTF-8" standalone="no"?>
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+  xmlns:ext="http://www.liquibase.org/xml/ns/dbchangelog-ext"
+  xmlns:pro="http://www.liquibase.org/xml/ns/pro"
+  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+  xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog-ext http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-ext.xsd
+                      http://www.liquibase.org/xml/ns/pro http://www.liquibase.org/xml/ns/pro/liquibase-pro-3.9.xsd
+                      http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-3.9.xsd">
+  <changeSet author="Payments team" id="1">
```
Comment (Kirsten1822) [inline changelog/db.changelog-2.6.xml:9]: Is there any reason why this isn't just your name?

**Comment by Kirsten1822** [comment]
Also is there anyway to raise the code coverage, as it's currently sitting at 70%?

File: changelog/db.changelog-2.6.xml (hunk 1 above)
Comment (johnwatson484) [reply changelog/db.changelog-2.6.xml]: Coz I copied and pasted it..

Fixed!

**Review by Kirsten1822 — APPROVED** [review]
Happy to approve :)

**Pull request merged**